.github/workflows/run-checker-ci.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           enable-trace,
           no-ts,
           no-ui,
+          enable-fips,
```

Comment:
This won't work - it will just add a new build with enable-fips. Just add the enable-fips on the already existing line with enable-trace as in `enable-fips enable-trace`.

Reply:
`enable-trace enable-fips` on line 26...